### ISSUE TITLE
Panning using the keyboard

### ISSUE THREAD

**Issue by stevenpetryk** [stevenpetryk/mafs#10]
Currently, users cannot pan using the keyboard. This is an easy fix, just something I neglected in the beta.

**Comment by stevenpetryk** [comment]
[Implemented in 0.7.2](https://github.com/stevenpetryk/mafs/releases/tag/v0.7.2).
